### PR TITLE
[master] Also target netstandard2.1 for Identity.EFCore

### PR DESCRIPTION
**Pull request by HaoK**
Fix for https://github.com/aspnet/AspNetCore/issues/13340

### CODE REVIEW

**Comment by HaoK** [comment]
@JunTaoLuo I tried to mimic the changes you made in https://github.com/aspnet/AspNetCore/commit/88a3a74dbb4fe04ac6265191a15d7c9a8013d70f#diff-d5cb1079ed5b07e65dcc675f6e4b7ef3  can you double check I did it correctly?

File: src/Identity/EntityFrameworkCore/src/Microsoft.AspNetCore.Identity.EntityFrameworkCore.csproj [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
   <PropertyGroup>
     <Description>ASP.NET Core Identity provider that uses Entity Framework Core.</Description>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFrameworks>netstandard2.1;netcoreapp3.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(DotNetBuildFromSource)' == 'true'">netcoreapp3.0</TargetFrameworks>
```
Comment (JunTaoLuo) [inline src/Identity/EntityFrameworkCore/src/Microsoft.AspNetCore.Identity.EntityFrameworkCore.csproj]: You don't need to do this, This package is not built in source build.

**Review by JunTaoLuo — APPROVED** [review]
Changes look good overall but you don't need the conditional for source build.

**Comment by JunTaoLuo** [comment]
Should we submit this for 3.0 ask mode? Either preview9 or post preview9? cc @Pilchie

**Comment by Pilchie** [comment]
Yes, as for which - is there a workaround for affected customers?  We don't have a lot of time to get things into Preview 9 builds at this point.

**Comment by HaoK** [comment]
I don't think there is a workaround for this

**Comment by Pilchie** [comment]
Okay - can you target the `release/3.0-preview9` branch and fill out the ask mode template.  I'm not sure it'll get approved for P9, but we can try.

**Comment by HaoK** [comment]
Rebase went poorly on this PR, created a new one for preview9 https://github.com/aspnet/AspNetCore/pull/13402

**Comment by HaoK** [comment]
I'll keep this PR around for master

**Pull request closed without merging**